### PR TITLE
ref(code-mappings): Remove feature flag for php auto code mappings (post-GA)

### PR DESCRIPTION
**Pull request by MichaelSun48**
Automatic code mapping support for PHP has been in GA for a week and has been stable. This removes the feature flag and any occurrences of it 

[Associated Sentry Options Automator PR](https://github.com/getsentry/sentry-options-automator/pull/979)
[Associated getsentry PR](https://github.com/getsentry/getsentry/pull/13440)

### CODE REVIEW

File: tests/sentry/tasks/test_derive_code_mappings.py [hunk 1]
```diff
@@ -344,21 +344,6 @@ def setUp(self):
...
-        repo_name = "php/place"
-        with patch(
-            "sentry.integrations.github.client.GitHubClientMixin.get_trees_for_org"
-        ) as mock_get_trees_for_org:
-            mock_get_trees_for_org.return_value = {
-                repo_name: RepoTree(Repo(repo_name, "master"), ["sentry/potato/kangaroo.php"])
-            }
-            derive_code_mappings(self.project.id, self.event_data)
-            # Check to make sure no code mappings were generated
-            assert not RepositoryProjectPathConfig.objects.exists()
```
Comment (MichaelSun48) [inline tests/sentry/tasks/test_derive_code_mappings.py:358]: Tests just if the flag is working - unnecessary now

File: src/sentry/tasks/derive_code_mappings.py [hunk 2]
```diff
@@ -102,12 +102,6 @@ def derive_code_mappings(
         logger.info("Event should not be processed.", extra=extra)
         return
 
-    # php automatic code mappings currently in LA
-    if data["platform"].startswith("php") and not features.has(
-        "organizations:derive-code-mappings-php", org
-    ):
-        return
-
```
Comment (vartec) [inline src/sentry/tasks/derive_code_mappings.py:110]: ideally this change would be in a separate PR, and the order of landing would be:
1. PR removing these lines 
2. Options automator PR
3. GetSentry PR
4. the rest of this PR cleaning up the flag. 

That being said, removing value from options-automator does not automatically remove it, so an OK way would be:
1. Options automator PR
2. GetSentry PR
3. This PR

**Review by vartec — APPROVED** [review]
